arm-operationalinsights/2015-03-20/swagger/OperationalInsights.json

```diff
@@ -0,0 +1,1110 @@
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SearchParameters"
+            },
+            "description": "The parameters required to execute a search query."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK response definition.",
+            "schema": {
+              "$ref": "#/definitions/SearchResultsResponse"
+            }
+          },
+          "202": {
+            "description": "Accepted response definition"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/search/{id}": {
+      "post": {
+        "tags": [
+          "Workspaces"
+        ],
+        "operationId": "Workspaces_UpdateSearchResults",
+        "description": "Gets updated search results for a given search query.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "workspaceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Log Analytics workspace name"
+          },
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The id of the search that will have results updated. You can get the id from the response of the response of the GetResults call."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK response definition.",
+            "schema": {
+              "$ref": "#/definitions/SearchResultsResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "LinkTarget": {
+      "properties": {
+        "customerId": {
+          "type": "string",
+          "description": "The GUID that uniquely identifies the workspace. "
+        },
+        "accountName": {
+          "type": "string",
+          "description": "The display name of the workspace.",
+          "x-ms-client-name": "DisplayName"
+        },
+        "workspaceName": {
+          "type": "string",
+          "description": "The DNS valid workspace name."
+        },
+        "location": {
+          "type": "string",
+          "description": "The location of the workspace."
+        }
+      },
+      "description": "Metadata for a workspace that isn't linked to an Azure subscription."
+    },
+    "Tag": {
+      "properties": {
+        "Name": {
```

Comment:
General comment: Are you sure that the model properties are in PascalCase? Please do not look at the server side code in C# as that can be misleading. Can you please confirm this by looking at the Fiddler/wire request/response ?
This is a general comment for all the model properties that are in PascalCase. The general convention is that all the json object properties should be in camelCase. JSON objects are like case sensitive dictionaries. Incorrect casing can have adverse effects in serialization and deserialization. The property will be ignored if the casing does not match during [de]serialization.